Benchmarks/Benchmarks/Signing/Signing.swift

```diff
@@ -0,0 +1,106 @@
+import Benchmark
+import Foundation
+import JWTKit
+
+let customThresholds = BenchmarkThresholds(
```

Comment:
I'd prefer this to be renamed to `thresholdTolerances`.
I talked with Joakim and he did confirm this is kind of a misnomer that the parameter is plainly named `threhsolds`.